addons/scopes/functions/fnc_adjustScope.sqf

```diff
@@ -36,6 +36,20 @@ private _zeroing = _adjustment select _weaponIndex;
 _zeroing params ["_elevation", "_windage", "_zero"];
 
 (GVAR(scopeAdjust) select _weaponIndex) params ["_maxVertical", "_verticalIncrement", "_maxHorizontal", "_horizontalIncrement"];
+
+if (_majorStep) then {
+    if(1/_verticalIncrement == floor(1/_verticalIncrement)) then {
```

Comment:
I think this needs explanation?

Reply:
`if(1%_verticalIncrement == 0) then`
Assuming, of course, that SQF even has modulo and that ^this is the correct syntax for it.

Reply:
SQF indeed has modulo (SQF is bad but not THAT bad, cmon').